src/pages/Goods/Detail/index.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { useGetProductDetail } from '@/api/hooks/useGetProductDetail';
+import { useGetProductOptions } from '@/api/hooks/useGetProductOptions';
+import { worker } from '@/mocks/browser';
+
+import { GoodsDetailPage } from './index';
+
+
+// useGetProductDetail, useGetProductOptions 모킹
+jest.mock('@src/api/hooks/useGetProductDetail', () => ({
+  useGetProductDetail: jest.fn(),
+}));
+
+jest.mock('@src/api/hooks/useGetProductOptions', () => ({
+  useGetProductOptions: jest.fn(),
```

Comment:
mocking path가 잘못된 거 같아요.

```suggestion
jest.mock('@/api/hooks/useGetProductDetail', () => ({
  useGetProductDetail: jest.fn(),
}));

jest.mock('@/api/hooks/useGetProductOptions', () => ({
```